xtask/src/codegen/mod.rs

```diff
@@ -53,6 +53,17 @@ pub struct Options {
     #[structopt(long)]
     libbpf_dir: PathBuf,
 
+    // sysroot options. Default to ubuntu headers installed by the
+    // libc6-dev-{arm64,armel}-cross packages.
```

Comment:
If you make this a doc comment it will show up when you run aya-gen --help. Could be useful to add docs to other flags too.